src/Component/HTML/ArticleList.purs

```diff
@@ -0,0 +1,74 @@
+module Component.HTML.ArticleList where
+
+import Prelude
+
+import Component.Classes as CC
+import Data.Article (Article)
+import Data.Author (profile, username)
+import Data.Formatter.DateTime (FormatterCommand(..), format)
+import Data.List (List, fromFoldable)
+import Data.Maybe (maybe)
+import Data.Profile (avatarToString)
+import Data.Username (toString)
+import Halogen.HTML as HH
+import Halogen.HTML.Properties as HP
+import Halogen.Themes.Bootstrap4 as HB
+import Network.RemoteData (RemoteData(..))
+
+articleList :: forall p i. RemoteData String (Array Article) -> HH.HTML p i
+articleList = case _ of
+  NotAsked -> HH.text "Not loaded"
+  Loading -> HH.text "Loading"
+  Failure err -> HH.text $ "Error loading articles: " <> err
+  Success articles -> HH.div_ $ articles <#> article
+
+article :: forall p i. Article -> HH.HTML p i
+article a =
+  HH.div
+  [ HP.class_ CC.articlePreview ]
+  [ HH.div
+    [ HP.class_ CC.articleMeta ]
+    [ HH.a_
+      [ HH.img
+        [ HP.src $ a.author # profile # _.avatar # maybe "" avatarToString
```

Comment:
I haven't done it yet, but the `Nothing` case of an avatar should fall back to some default profile picture present for all users. For example, we could pick an auto-gen one from GitHub or something and save it as an asset, then link to that instead of providing the empty string here. Perhaps there should be an additional function:

```purescript
avatar :: Maybe Avatar -> String
avatar Nothing = "default-photo.png"
avatar (Just s) = s
```